app/controllers/api/v0/pages_controller.rb

```diff
@@ -27,8 +27,10 @@ def index
           .includes(:versions)
           .order('versions.capture_time')
         results.as_json(include: :versions)
-      else
+      elsif should_include_latest
```

Comment:
I will never get used to this feature of Ruby syntax. Is it really that hard to type 'e'? End of rant.

Reply:
This, coming from a Python person, where that 's' is too hard to type, too? ;)